README.md

```diff
@@ -109,12 +108,6 @@ except ValueError as error :
     print("Connector write failed", error)
 ```
 
-### Specifying the isolation level
```

Comment:
Why is this  being deleted? Looks like you might not have pulled Arvind's changes? He made a PR adding and updating information on the readME

Reply:
@rajmera3 Apologies my file sync on OneDrive messed it up. My last commit (commit a7cf3195274f2a790c000485b74d3ae5141c22e7) has the updated README from previous commits including Arvind's.